src/components/new-request/name-check/name-check.vue

```diff
@@ -1013,6 +1013,11 @@ export default class NameCheck extends Vue {
   transition: none !important;
 }
 
+::v-deep .theme--light.v-input--is-disabled input, .theme--light.v-input--is-disabled textarea {
+  background-color: #ffffff;
+  color: $gray9;
```

Comment:
Is there no `readonly` prop on the v-input that you could use (instead of `disabled`)?

Also, you can use "white" for the bg colour.

Reply:
If it's more work to style the readonly version, I'm OK with it like this.

Also, please add a code comment explaining what you're doing with this v-deep override.

Reply:
> Is there no `readonly` prop on the v-input that you could use (instead of `disabled`)?

Yes, no readonly. Because for the readonly version, I also need to add the dotted underline, but for the disabled version, it already comes with

Reply:
OK, fine by me.

Reply:
> Also, please add a code comment explaining what you're doing with this v-deep override.

done